osu.Server.Queues.ScoreStatisticsProcessor.Tests/PlayCountProcessorTests.cs

```diff
@@ -19,6 +21,27 @@ public void TestPlaycountIncreaseMania()
             WaitForDatabaseState("SELECT playcount FROM osu_user_stats_mania WHERE user_id = 2", 2, CancellationToken);
         }
 
+        [Fact]
+        public void TestGlobalPlaycountsIncrement()
+        {
+            const int attempt_count = 100;
+
+            AddBeatmap();
+
+            Task.Run(() =>
```

Comment:
Is there a reason why this is running in a Task that doesn't seem to be awaited anywhere? It almost looks like this task could still be running even after this test is complete, is that not the case?

Fwiw, I believe xunit tests can be `public async Task`

Reply:
Yeah that's probably bad, will fix.